.github/workflows/build-and-test-pr.yml

```diff
@@ -54,7 +54,7 @@ jobs:
     name: "Test Mobile E2E"
     needs: determine-affected
     if: ${{contains(needs.determine-affected.outputs.paths, 'ledger-live-mobile') && !github.event.pull_request.head.repo.fork}}
-    uses: LedgerHQ/ledger-live/.github/workflows/test-mobile-e2e-reusable.yml@develop
+    uses: LedgerHQ/ledger-live/.github/workflows/test-mobile-e2e-reusable.yml@support/qaa_312_detox_speculos_add_accounts
```

Comment:
```suggestion
    uses: LedgerHQ/ledger-live/.github/workflows/test-mobile-e2e-reusable.yml@develop
```

Reply:
I am just waiting for the checks to pass before reverting last commit